.code-samples.meilisearch.yaml

```diff
@@ -19,15 +19,15 @@ get_one_document_1: |-
 get_documents_1: |-
   let documents: Vec<Movie> = movies.get_documents(None, Some(2), None).await.unwrap();
 add_or_replace_documents_1: |-
-  let progress: Progress = movies.add_or_replace(&[
+  let progress: Progress = movies.add_or_replace(vec!(
```

Comment:
Please revert all the changes of the code samples. It must still work

Reply:
Just restored the old document (the current one in the main branch).